src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -21,6 +21,7 @@
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <CopyNuGetImplementations>true</CopyNuGetImplementations>
+    <DeployExtension Condition="'$(VisualStudioVersion)' == '14.0'">false</DeployExtension>
```

Comment:
Nit: more resilient to use this version:

``` xml
<DeployExtension Condition="'$(VisualStudioVersion)' == '15.0'">true</DeployExtension>
```

Accounts for the case where `VisualStudioVersion` is undefined.